### PR TITLE
Hedingfield updates

### PR DESCRIPTION
**Pull request by HEdingfield**

### CODE REVIEW

File: tidegates/utils.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def __init__(self, name):
         self.name = name
 
     def __enter__(self):
-        if arcpy.CheckExtension(self.name) == "Available":
+        if arcpy.CheckExtension(self.name) == u"Available":
```
Comment (HEdingfield) [inline tidegates/utils.py]: Unsure if this particular unicode typing was necessary.

**Comment by phobson** [comment]
All tests from this branch pass on my system. Looks :+1: 

```
F:\phobson\Tidegates\python-tidegates [master]> git checkout hedingfield-updates
Switched to branch 'hedingfield-updates'
Your branch is up-to-date with 'hylton/hedingfield-updates'.
F:\phobson\Tidegates\python-tidegates [hedingfield-updates]> nosetests --with-coverage --cover-package=tidegates
.........................................................................................
Name                     Stmts   Miss  Cover   Missing
------------------------------------------------------
tidegates.py                 3      0   100%
tidegates\tidegates.py      43     13    70%   156-165, 169-177
tidegates\toolbox.py       126     34    73%   256-275, 308-327
tidegates\utils.py         153      0   100%
------------------------------------------------------
TOTAL                      325     47    86%
----------------------------------------------------------------------
Ran 89 tests in 13.076s
```

File: tidegates/utils.py (hunk 1 above)
Comment (phobson) [inline tidegates/utils.py]: well it doesn't break anything. I'll keep it.

File: tidegates/testing/helpers.py [hunk 2]
```diff
@@ -70,7 +68,7 @@ class NoseWrapper(nptest.Tester):
 
 
     def test(self, label='fast', verbose=1, with_id=True, exe=True,
-             doctests=False, coverage=False, packageinfo=True, extra_argv=[],
+             doctests=False, coverage=False, packageinfo=True, extra_argv=None,
```
Comment (phobson) [inline tidegates/testing/helpers.py]: The numpy nose wrapper requires this to be a list. If you're going to change it to `None`, you need to check for that and set it to an empty list before it gets used.

**Pull request closed without merging**